chromadb/test/segment/test_vector.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 from typing import Generator, List, Callable, Iterator, Type, cast
 from chromadb.config import System, Settings
+from chromadb.segment.impl.metadata.sqlite import SqliteMetadataSegment
```

Comment:
Nit: unused please remove

Reply:
I noticed this but didn't highlight it as unused (some Pycharm bug). Let me update